src/components/customizableAside.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import CustomizableAvatar from "./components/common/customavatar/CustomizableAvatar"
+
+function CustomizableAside({asideCustomClass, src, height, width, pContent, h2Content}) {
```

Comment:
Inside your `<CustomizableAside>` component, you make use of the `alt` attribute in `<CustomizableAvatar>` but you do not pass `alt` as one of the props.